src/Akeneo/Category/back/Infrastructure/Storage/Sql/GetCategory.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Category\Infrastructure\Storage\Sql;
+
+use Akeneo\Category\Domain\Query\GetCategoryInterface;
+use Akeneo\Category\Domain\Model\Category;
+use Akeneo\Category\Domain\ValueObject\CategoryId;
+use Akeneo\Category\Domain\ValueObject\Code;
+use Akeneo\Category\Domain\ValueObject\LabelCollection;
+use Doctrine\DBAL\Connection;
+
+/**
+ * @copyright 2022 Akeneo SAS (https://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class GetCategory implements GetCategoryInterface
```

Comment:
```suggestion
class GetCategorySql implements GetCategoryInterface
```
? Pour rester cohérent avec le nommage de nos classes dans Infrastructure. Et pour bien connoté la classe comme étant du sql.